File: AppGenerateFiles/dbo/Model/Detalle_Factura_Recibo.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Detalle_Factura_Recibo : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? id { get; set; }
       public int? id_factura { get; set; }
       public int? id_cuota { get; set; }
       public Double? total_cuota { get; set; }
       public Double? monto_pagado { get; set; }
       public Double? capital_restante { get; set; }
       public string? concepto { get; set; }
       public Double? tasa_cambio { get; set; }
       [ManyToOne(TableName = "Tbl_Cuotas", KeyColumn = "id_cuota", ForeignKeyColumn = "id_cuota")]
       public Tbl_Cuotas? Tbl_Cuotas { get; set; }
       [ManyToOne(TableName = "Transaccion_Factura", KeyColumn = "id_factura", ForeignKeyColumn = "id_factura")]
       public Transaccion_Factura? Transaccion_Factura { get; set; }
   }
}

File: AppGenerateFiles/dbo/FrontModel/Detalle_Factura_Recibo.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../../WDevCore/WModules/WComponentsTools.js";
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
import { Tbl_Cuotas }  from './Tbl_Cuotas.js'
import { Transaccion_Factura }  from './Transaccion_Factura.js'
class Detalle_Factura_Recibo extends EntityClass {
   constructor(props) {
       super(props, 'EntityDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ id;
   /**@type {Number}*/ total_cuota;
   /**@type {Number}*/ monto_pagado;
   /**@type {Number}*/ capital_restante;
   /**@type {String}*/ concepto;
   /**@type {Number}*/ tasa_cambio;
   /**@type {Tbl_Cuotas} ManyToOne*/ Tbl_Cuotas;
   /**@type {Transaccion_Factura} ManyToOne*/ Transaccion_Factura;
}
export { Detalle_Factura_Recibo }
